sdl2-vga-terminal/src/VgaTerminal.hpp

```diff
@@ -44,19 +44,26 @@ class VgaTerminal : public Window
         CURSOR_MODE_VERTICAL = 3,
     };
     static constexpr uint8_t NUM_CURSOR_MODES = 4;
-
     CURSOR_MODE cursor_mode = CURSOR_MODE::CURSOR_MODE_NORMAL;
 
+    enum class CURSOR_SPEED : uint16_t {
+        CURSOR_SPEED_NORMAL = 500,
+        CURSOR_SPEED_FAST = 250,
+        CURSOR_SPEED_SLOW = 750
+    };
+    static constexpr uint8_t NUM_CURSOR_SPEEDS = 3;
+
     static const std::string getVersion();
 
     VgaTerminal() = delete;
     VgaTerminal(const VgaTerminal&) = delete;
     VgaTerminal(const VgaTerminal&&) = delete;
     VgaTerminal& operator=(const VgaTerminal&) = delete;
     VgaTerminal& operator=(const VgaTerminal&&) = delete;
-    VgaTerminal(const std::string &title, const int winFlags, const int drvIndex, const int renFlags);
-    VgaTerminal(const std::string &title, const int width, const int height, const int winFlags, const int drvIndex, const int renFlags);
-    VgaTerminal(const std::string &title, const int x, const int y, const int width, const int height, const int winFlags, const int drvIndex, const int renFlags);
+    VgaTerminal(const std::string& title);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'VgaTerminal' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5472420)